compiler-rt/lib/builtins/README.txt

```diff
@@ -137,11 +137,12 @@ si_int __ucmpti2(tu_int a, tu_int b);
 di_int __fixsfdi(      float a);
 di_int __fixdfdi(     double a);
 di_int __fixxfdi(long double a);
+di_int __fixtfdi(long double a);
 
 ti_int __fixsfti(      float a);
 ti_int __fixdfti(     double a);
 ti_int __fixxfti(long double a);
-uint64_t __fixtfdi(long double input);  // ppc only, doesn't match documentation
+ti_int __fixtfti(long double a);
 
 su_int __fixunssfsi(      float a);
 su_int __fixunsdfsi(     double a);
```

Comment:
I think this change is going to enable bunch of other functions as well that are not documented here. Eg: fixunstfsi, etc. Should we document these as well?

Reply:
Yes it probably makes sense to add these too.

Reply:
Done in the latest version.